apps/dashboard/app/models/batch_connect/session_context.rb

```diff
@@ -59,6 +59,37 @@ def method_missing(method_name, *arguments, &block)
     # @return [Boolean]
     def respond_to_missing?(method_name, include_private = false)
       /^(?<id>[^=]+)=?$/ =~ method_name.to_s && self[id] || super
+   end
+    
+
+    # @return [Boolean]
+    def attribute_cache_enabled?
+      self.any? {|v| v.opts[:cacheable]  } 
+    end
+    
+    
+    # @return [Boolean]   
+    def app_specific_cache_enabled?
+      self[:cacheable]
+    end
+   
+     # @return [Boolean]   
+    def global_cache_enabled? 
+       ENV["OOD_BATCH_CONNECT_CACHE_ATTR_VALUES"]
```

Comment:
Move this to the ConfigurationSingleton, use its to_bool method, and provide a default to true. Then you can reference it here using `Configuration.batch_connect_global_cache_enabled?` (or whatever the method is named)

Reply:
Ahh yes

Reply:
using to_bool will equate nil with false. Is this behavior wanted on on the global configurations